src/main/java/com/ita/if103java/ims/controller/UserController.java

```diff
@@ -0,0 +1,94 @@
+package com.ita.if103java.ims.controller;
+
+
+import com.ita.if103java.ims.dto.UserDto;
+import com.ita.if103java.ims.dto.transfer.ExistData;
+import com.ita.if103java.ims.entity.User;
+import com.ita.if103java.ims.mapper.UserDtoMapper;
+import com.ita.if103java.ims.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+
+    private UserService userService;
+    private UserDtoMapper userDtoMapper;
+
+    @Autowired
+    public UserController(UserService userService, UserDtoMapper userDtoMapper) {
+        this.userService = userService;
+        this.userDtoMapper = userDtoMapper;
+    }
+
+    @GetMapping(value = "/get/{id}")
```

Comment:
I would recommended to avoid specifying method type exactly in url
like `GET` -> `/users/get/{id}`

Instead of this we can to follow some `REST` rules:

|HTTP method| url | Description        | 
|------------------|------|----------------------|
| GET        | /users/{id}       | Find user by id  |
| GET        | /users              | Find all users    |
| GET        | users/me         | Find info about current user |
| PUT        | users/me         | Update info about current user |
| DELETE | users/{id}         | Delete user by ID                |
| GET        | account/users | Find users related to our `account` |

Reply:
Agree with Rostyslav